src/wp-content/themes/twentytwentytwo/inc/block-patterns.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Twenty Twenty-Two: Block Patterns
+ *
+ * @since Twenty Twenty-Two 1.0
+ */
+
+if ( ! function_exists( 'twentytwentytwo_register_block_patterns' ) ) :
+	function twentytwentytwo_register_block_patterns() {
```

Comment:
```suggestion
	/**
	 * Registers block patterns and categories.
	 *
	 * @since Twenty Twenty-Two 1.0
	 */
	function twentytwentytwo_register_block_patterns() {
```